src/main/java/uk/gov/cslearning/record/service/catalogue/LearningCatalogueService.java

```diff
@@ -50,9 +50,13 @@ public LearningCatalogueService(RestTemplate restTemplate, RequestEntityFactory
 
     @Cacheable("courseId")
     public Course getCourse(String courseId) {
-        RequestEntity requestEntity = requestEntityFactory.createGetRequest(String.format(courseUrlFormat, courseId));
-        ResponseEntity<Course> responseEntity = restTemplate.exchange(requestEntity, Course.class);
-        return responseEntity.getBody();
+        try {
+            RequestEntity requestEntity = requestEntityFactory.createGetRequest(String.format(courseUrlFormat, courseId));
+            ResponseEntity<Course> responseEntity = restTemplate.exchange(requestEntity, Course.class);
+            return responseEntity.getBody();
+        } catch (Exception e) {
```

Comment:
We should be specific about what kind of Exception we expect here. We should also at least log the exception so it doesn't fail silently.